app/src/main/java/com/connorlay/knitgrid/models/Pattern.java

```diff
@@ -0,0 +1,101 @@
+package com.connorlay.knitgrid.models;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.orm.SugarRecord;
+
+import java.util.List;
+
+/**
+ * Created by connorlay on 3/17/16.
+ */
+public class Pattern extends SugarRecord implements Parcelable {
+    private String name;
+    private int rows;
+    private int cols;
+    private boolean alternates;
```

Comment:
I don't like this name. I would use alternating, and then use 'alternating' as well for both the getter 'isAlternating' and the setter 'setAlternating'.